src/main/java/org/spdx/storage/simple/InMemSpdxStore.java

```diff
@@ -414,7 +414,7 @@ public void leaveCriticalSection(IModelStoreLock lock) {
 	}
 
 	@Override
-	public Optional<String> getCaseSensisitiveId(String nameSpace, String caseInsensisitiveId) {
+	public Optional<String> getCaseSensitiveId(String nameSpace, String caseInsensisitiveId) {
 		Objects.requireNonNull(nameSpace, "Namespace can not be null");
 		Objects.requireNonNull(caseInsensisitiveId, "CaseInsensisitiveId can not be null");
 		String objectUri = nameSpace + "#" + caseInsensisitiveId;
```

Comment:
```suggestion
	public Optional<String> getCaseSensitiveId(String nameSpace, String caseInsensitiveId) {
		Objects.requireNonNull(nameSpace, "Namespace can not be null");
		Objects.requireNonNull(caseInsensitiveId, "CaseInsensitiveId can not be null");
		String objectUri = nameSpace + "#" + caseInsensitiveId;
```

The second parameter for `getCaseSensitiveId()` is `caseInsensitiveId`.